reconstruction/cvt/src/main/java/org/jlab/rec/cvt/services/TracksFromTargetRec.java

```diff
@@ -301,20 +302,20 @@ public List<Track> getTracks(DataEvent event, boolean initFromMc, boolean kfFilt
                         fittedTrack = recUtil.recovTrkMisClusSearch(seed, hlx, cov, kf2, kf, pid, surfaces, xb, yb,
                             this.SVTclusters, this.SVTcrosses, 
                             swimmer, solenoidScale, solenoidValue, measure);
-                        if(fittedTrack!=null) {
-                            fittedTrack.setStatus(1); 
-                            if(Constants.getInstance().seedingDebugMode) 
-                                System.out.println("RECOVERED..."+fittedTrack.toString());
-                        } 
+//                        if(fittedTrack!=null) {
```

Comment:
I'm not certain but it seems to me there are still instances in which track status may not be set correctly or consistently. Wouldn't be safer to:
- initialize status=-1 in the Track class;
- set the status in the Track constructor Track(seed, kf, pid) based on the kf results
?
In this way it would be done in a single place and it would be easier to ensure consistency